omc3/utils/stats.py

```diff
@@ -48,6 +48,14 @@ def circular_mean(data, period=PI2, errors=None, axis=None):
     return np.angle(np.average(np.exp(phases), axis=axis, weights=weights)) * period / PI2
 
 
+def circular_nanmean(data, period=PI2, errors=None, axis=None):
```

Comment:
why do we have NaNs?

Reply:
in the way the crdts are processed there is a (slight) chance to encounter Nan. 
e.g if you analyze 3 files and there is one BPM missing in one file while present in the other two, in the average over these there would be a Nan for this BPM in one file
data from the other could still be good, so averaging just takes these two val then